### DISCUSSION TITLE
Conflict between the NISTHA data collected from NCERT and current version of cQube(5.0.3)

### DISCUSSION THREAD

**Discussion by Mohanapriya777** [Sunbird-cQube/community#825] [General]
Issue 1: Two columns are missing in 3 NISHTHA files: district_code, district_name 

1. Absence of these columns is causing issues during data ingestion
2. File having missing columns:  NISHTHA_program_implementation_status, NISHTHA_medium_of_instructions, NISHTHA_Program-wise_achieved_certificates

Issue 2: Two files are missing in the NISHTHA data

1. Absence of these files is not allowing us to visualize all data from the NISHTHA program
2. NISHTHA Files missing from NCERT data: NISHTHA_participants_per_program, NISHTHA_course-wise_certification

Issue 3: It is observed that Assam Schema is much different than the Puducherry schema for the same cQube version - 5.0.3

1. It's not uncommon for software versions to exhibit differences or receive updates even within the same stable release, such as version 5.0.3. Kindly share the possibilities.
2. The following files are available in Assam NISHTHA schema: diksha_nishtha_consumption-by-district, diksha_nishtha_consumption-by-district, diksha_nishtha_percentage-enrollment-certification, diksha_nishtha_program-started, diksha_nishtha_tot-courses-medium.

**Comment by rhwarrier** [comment]
Regarding NISHTHA programs, Naman has clarified that if district level data has to be included, the state has to source the data - NCERT does not seem to have this available. If the state too does not have the data, we can follow the NVSK schema, but this will require code changes to be carried out. 
Please refer : https://cqube.sunbird.org/use-cqube/ingesting-the-data/steps-to-ingest-schema
The better option here is to source district level data, as this is what the state will find useful as well. State level aggregates will not help the state take any actions. 

Assam uses an older schema version (NISHTHA 4). Puducherry has NISHTHA 5 - these changes have come in from the NISHTHA team based on the visualisations and reports they want. What we're seeing for Puducherry is the latest requirement as per NCERT
Please see https://cqube.sunbird.org/use-cqube/getting-started-1/all-cqube-schemas-used-for-vsk
